ducc/lib/cvmfs.go

```diff
@@ -520,3 +520,24 @@ func RepositoryExists(CVMFSRepo string) bool {
 		return false
 	}
 }
+
+//writes data to file and ingest in cvmfs repo path
```

Comment:
Rather "publish" than "ingest" since the ingestion term is charged with publishing tarballs.